test/controllers/TransferAgreementControllerSpec.scala

```diff
@@ -159,13 +170,16 @@ class TransferAgreementControllerSpec extends FrontEndTestHelper {
       val dataString: String = data.asJson.printWith(Printer(dropNullValues = false, ""))
       wiremockServer.stubFor(post(urlEqualTo("/graphql"))
         .willReturn(okJson(dataString)))
+      val incompleteTransferAgreementForm: Seq[(String, String)] = getTransferAgreementForm(6)
 
```

Comment:
Would it be better here to just have `Seq()` instead of `getTransferAgreementForm(6)` ? This working is dependant on there being 6 fields on the form which could change (although you could just change the tests) and it's not immediately obvious that this is an empty form. What do you think?

Reply:
😄  Ironically, I did this because I thought it would be clearer what was happening since it's consistent with the other tests. But yeah, I see what you mean, it is weird to pass in 6 only to have the code drop those values and return an empty `Seq`. Also, by making it a `Seq()`, you can see that it's empty immediately and the variable name gives additional info as to why it's an empty `Seq`